src/components/WorkPoolQueuesTable.vue

```diff
@@ -3,9 +3,14 @@
     <p-list-header sticky>
       <ResultsCount v-if="selected.length == 0" label="Work Queue" :count="filteredWorkPoolQueues.length" />
       <SelectedCount v-else :count="selected.length" />
-      <p-button v-if="can.create.work_queue && !selected.length" size="sm" icon="PlusIcon" :to="routes.workPoolQueueCreate(workPoolName)" />
 
-      <WorkPoolQueuesDeleteButton v-if="can.delete.work_queue" :work-pool-name="workPoolName" :work-pool-queues="selected" @delete="handleDelete" />
+      <template v-if="workPool?.can.update && !selected.length">
+        <p-button size="sm" icon="PlusIcon" :to="routes.workPoolQueueCreate(workPoolName)" />
+      </template>
+
+      <template v-if="workPool?.can.delete">
```

Comment:
was thinking that these seemed potentially confusing at a glance that creating a work queue is tied to workPool.can.update and deleting a work queue is tied to workPool.can.delete. 


but then on second thought... i think it shouldn't make a difference right b/c both are ultimately `manage` level permissions?

Reply:
Hmm yeah now that I'm thinking about it the delete should probably be `update` because you're updating the work queues? But I think you're right they map to manage either way.

Reply:
Update in https://github.com/PrefectHQ/prefect-ui-library/pull/2689/commits/886424b286d970535e76231c03e8e875c322b3ed